.github/workflows/iroha2-no-incorrect-image.yml

```
@@ -0,0 +1,25 @@
+name: I2::CI::check_for_incorrect_images
+
+on:
+  push:
+    branches:
+      - iroha2-dev
+      - iroha2-stable
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.11"
+      - uses: actions/checkout@v3
+      - name: Install depenencies
+        run: pip install -r .github/scripts/ci_test/requirements.txt --no-input
+      - name: Check containers on iroha2-stable branch
+        if: github.ref == 'refs/heads/iroha2-stable'
+        run: python .github/scripts/ci_test/ci_image_scan.py --allow iroha2:stable -- docker-compose.dev.local.yml docker-compose.dev.single.yml docker-compose.dev.yml docker-compose.single.yml docker-compose.stable.single.yml docker-compose.stable.yml docker-compose.yml
+      - name: Check containers on iroha2-dev branch
+        if: github.ref == 'refs/heads/iroha2-dev'
+        run: python .github/scripts/ci_test/ci_image_scan.py --allow iroha2:dev iroha2:stable -- docker-compose.dev.local.yml docker-compose.dev.single.yml docker-compose.dev.yml docker-compose.single.yml docker-compose.stable.single.yml docker-compose.stable.yml docker-compose.yml
```

```suggestion
        run: python .github/scripts/ci_test/ci_image_scan.py --allow iroha2:stable -- docker-compose.stable*.yml
      - name: Check containers on iroha2-dev branch
        if: github.ref == 'refs/heads/iroha2-dev'
        run: python .github/scripts/ci_test/ci_image_scan.py --allow iroha2:dev -- docker-compose.dev*.yml
```

That is how I imagined it to work.

---

Understood, I'll fix it to pick between the globs and filenames very soon